src/menu/views/cpakfs_manager.c

```diff
@@ -0,0 +1,1016 @@
+            cpakfs_path_strings[9].ext,
+            cpakfs_path_strings[10].ext,
+            cpakfs_path_strings[11].ext,
+            cpakfs_path_strings[12].ext,
+            cpakfs_path_strings[13].ext,
+            cpakfs_path_strings[14].ext,
+            cpakfs_path_strings[15].ext
+        );
+
+        ui_components_main_text_draw(style,
+            ALIGN_LEFT, VALIGN_TOP,
+            "\n"
+            "\n"
+            "\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n"
+            "                                          %s\n", 
+            controller_pak_name_notes_bank_size[0],
+            controller_pak_name_notes_bank_size[1],
+            controller_pak_name_notes_bank_size[2],
+            controller_pak_name_notes_bank_size[3],
+            controller_pak_name_notes_bank_size[4],
+            controller_pak_name_notes_bank_size[5],
+            controller_pak_name_notes_bank_size[6],
+            controller_pak_name_notes_bank_size[7],
+            controller_pak_name_notes_bank_size[8],
+            controller_pak_name_notes_bank_size[9],
+            controller_pak_name_notes_bank_size[10],
+            controller_pak_name_notes_bank_size[11],
+            controller_pak_name_notes_bank_size[12],
+            controller_pak_name_notes_bank_size[13],
+            controller_pak_name_notes_bank_size[14],
+            controller_pak_name_notes_bank_size[15]
+        );
+    }
+
+    ui_components_context_menu_draw(&options_context_menu);
+
+
+    style = (has_mem && !corrupted_pak) ? STL_DEFAULT : STL_GRAY;
+
+
+    if (!use_rtc) {
+        ui_components_main_text_draw(STL_ORANGE,
+            ALIGN_LEFT, VALIGN_TOP,
+            "No RTC\n"
+        );
+        style = STL_GRAY;
+    }
+
+
+
+    // Actions bars
+
+    if (!corrupted_pak) {
+
+        ui_components_actions_bar_text_draw(style,
+            ALIGN_LEFT, VALIGN_TOP,
+            "A: Dump Pak\n"
+            "L|Z: Dump single Note\n"
+        );
+        ui_components_actions_bar_text_draw(style,
+            ALIGN_RIGHT, VALIGN_TOP,
+            "START: Restore Pak\n"
```

Comment:
This only displays a message to state that you need to do it from a file. Maybe move the `L|Z` context to this place and move `B` context under `A`. the "Dump" context could be moved to the centre, but could also just be ommited and rely on documentation?